test_requirements.txt

```diff
@@ -6,7 +6,7 @@ pytest-xdist>=1.14,<2.0.0
 pytest-mock>=1.5.0,<3.0.0
 mock>=1.0.1,<5.0.0
 coverage>=3.7.1,<6.0.0
-scikit-learn<1.0.0
+scikit-learn
```

Comment:
please remove scikit-learn from test requirements. Place it in regular requirements if needed. It has nothing to do with testing